__tests__/fixture/source.ts

```diff
@@ -79,3 +85,18 @@ export const sourcesFixture: DeepPartial<MachineSource>[] = [
     private: false,
   },
 ];
+
+export const squadFixture: DeepPartial<SquadSource>[] = [
```

Comment:
I would just define this in the file you are using it, we want to move away from fixtures probably since they often create flaky tests